src/ui/server/helpers/mappings/map-applications/index.ts

```diff
@@ -17,7 +18,7 @@ export const mapApplication = (application: Application) => {
     lastUpdated: formatDate(new Date(updatedAt)),
     referenceNumber,
     buyerLocation: buyer?.country?.name || DEFAULT.EMPTY,
-    buyerName: buyer.companyOrOrganisationName || DEFAULT.EMPTY,
+    buyerName: buyer.companyOrOrganisationName ? replaceCharacterCodesWithCharacters(buyer.companyOrOrganisationName) : DEFAULT.EMPTY,
```

Comment:
could this not be `replaceCharacterCodesWithCharacters(buyer.companyOrOrganisationName) ?? DEFAULT.EMPTY`

Reply:
Great spot, it can now because the character code helper function param is optional. I've changed to

```js
replaceCharacterCodesWithCharacters(buyer.companyOrOrganisationName) || DEFAULT.EMPTY
```